### PR TITLE
fix:Edit the Job page icon offset

### PR DESCRIPTION
**Pull request by MapleWithoutWords**
image

### CODE REVIEW

File: src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor [hunk 1]
```diff
@@ -213,7 +213,7 @@
                                         </MRow>
                                     </AutoHeightContent>
                                     <FooterContent>
-                                        <div class="d-flex mt-9">
+                                                <div class="d-flex mt-9" style="align-items: center;">
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor:216]: align-center    Please try to use built-in css instead of style

**Review by wzh425 — COMMENTED** [review]
Please try to use built-in css instead of style

File: src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor (hunk 1 above)
Comment (MapleWithoutWords) [reply src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor:216]: this is flex layout. align-center not effect. must use align-items

Comment (MapleWithoutWords) [reply src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor:216]: image

Comment (wzh425) [reply src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor:216]: Align-center is the built-in css of masa blazer. Its effect is to align items: center   @MapleWithoutWords

Comment (MapleWithoutWords) [reply src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor]: > Align-center is the built-in css of masa blazer. Its effect is to align items: center @MapleWithoutWords

ok,already changed.

**Review by wzh425 — APPROVED** [review]
LGTM

**Pull request merged**